google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/it/ITPubSubTest.java

```diff
@@ -78,8 +77,11 @@ static MessageAndConsumer create(PubsubMessage message, AckReplyConsumer consume
 
   @BeforeClass
   public static void setupClass() throws Exception {
-    topicAdminClient = TopicAdminClient.create();
-    subscriptionAdminClient = SubscriptionAdminClient.create();
+    topicAdminClient =
+        TopicAdminClient.create(TopicAdminSettings.newBuilder().setEndpoint(endPoint).build());
+    subscriptionAdminClient =
+        SubscriptionAdminClient.create(
+            SubscriptionAdminSettings.newBuilder().setEndpoint(endPoint).build());
```

Comment:
removing endpoint. I wanted to test the integration test against loadtest endpoint as ESF config changes are available there. I thought it will be good to have the flexibility to modify endpoints, but then keeping in mind that this is public repo we need not worry about that.